util/minetestmapper.py

```diff
@@ -190,22 +194,126 @@ def usage():
         draworigin = True
     elif o == "--drawunderground":
         drawunderground = True
+    elif o == "--geometry":
+        geometry_string = a
+    elif o == "--region":
+        region_string = a
     else:
         assert False, "unhandled option"
 
+nonchunky_xmin = -1500
+nonchunky_xmax = 1500
+nonchunky_zmin = -1500
+nonchunky_zmax = 1500
+
+if geometry_string is not None:
+    parts = geometry_string.split("+")
+    if len(parts) == 3:
+        coords_string, width_string, height_string = parts
+        coords = coords_string.split(":")
+        if len(coords) == 2:
+            x_string, z_string = coords
+            x = int(x_string)
+            z = int(z_string)
+            this_width = int(width_string)
+            this_height = int(height_string)
+            nonchunky_xmin = x
+            nonchunky_xmax = nonchunky_xmin + this_width - 1  # inclusive rect
+            nonchunky_zmin = z
+            nonchunky_zmax = nonchunky_zmin + this_height - 1  # inclusive rect
+            print("#geometry:")
```

Comment:
or
```
            print("#geometry:\n" + \
                  "#  x:{}\n".format(x) + \
                  "#  z:{}\n".format(z) + \
                  "#  width:{}\n".format(this_width) + \
                  "#  height:{}\n".format(this_height) + \
                  "region:\n" + \
                  "  xmin:{}\n".format(nonchunky_xmin) + \
                  "  xmax:{}\n".format(nonchunky_xmax) + \
                  "  zmin:{}\n".format(nonchunky_zmin) + \
                  "  zmax:{}\n".format(nonchunky_zmax))
```
or
```python
            print("#geometry:\n"
                  "#  x:{}\n"
                  "#  z:{}\n"
                  "#  width:{}\n"
                  "#  height:{}\n"
                  "region:\n"
                  "  xmin:{}\n"
                  "  xmax:{}\n"
                  "  zmin:{}\n"
                  "  zmax:{}\n".format(x,z,this_width,this_height,
                                       nonchunky_xmin,nonchunky_xmax,
                                       nonchunky_zmin,nonchunky_zmax))
```
there are so many ways, but that last one seems the most readable and keeps indentation neat.